inst/templates/tidy_contributing.md

```diff
@@ -0,0 +1,156 @@
+# Contributing to {{{package}}}
+
+-   [Prerequisites](#prerequisites)
+-   [Package mechanics](#package-mechanics)
+    -   [Fork, clone, branch](#fork-clone-branch)
+    -   [Establish a baseline](#establish-a-baseline)
+    -   [Style](#style)
+    -   [Documentation](#documentation)
+    -   [Testing](#testing)
+    -   [NEWS](#news)
+    -   [Committing your changes](#commiting-your-changes)
+-   [Making the pull request](#making-the-pull-request)
+    -   [Review, revise, repeat](#review-revise-repeat)
+-   [Resources](#resources)
+-   [Code of Conduct](#code-of-conduct)
+
+This explains how to propose a change to {{{package}}} via a pull request using
+Git and Github. 
+
+For more general info about contributing to the tidyverse, see the 
+[Resources](#resources) at the end of this document.
+
+## Prerequisites
+
+Before you do a pull request, you should always file an issue and make sure
+someone from the tidyverse team agrees that it’s a problem, and is happy with
+your basic proposal for fixing it. If you’ve found a bug, first create a minimal
+[reprex](https://www.tidyverse.org/help/#reprex).
+
+## Package mechanics
+
+### Fork, clone, branch
+
+The first thing you'll need to do is to fork the 
+[{{{package}}} GitHub repo](https://github.com/tidyverse/{{{package}}}), and 
+then clone it locally. We recommend that you create a branch for each PR.
+
+### Establish a baseline
```

Comment:
Maybe just "Check". Could also mention that you need to make sure it passes after your change as well.

Reply:
I mention R CMD check at the end of the testing section, but, since NEWS now comes after the Test section, maybe I should move it elsewhere?

Reply:
_You'll do this again _after_ you've made changes, but it's best to establish a baseline._